libraries/Microsoft.Bot.Builder.Ai.LUIS/LuisRecognizer.cs

```diff
@@ -66,26 +53,18 @@ public Task<(RecognizerResult recognizerResult, LuisResult luisResult)> CallAndR
             return Recognize(luisRequest, ct, _luisRecognizerOptions.Verbose);
         }
 
-        /// <summary>
-        /// Runs an utterance through a LUIS recognizer and returns the recognizer results.
-        /// </summary>
-        /// <param name="request">A LUIS request for an utterance.</param>
-        /// <param name="ct">A cancellation token.</param>
-        /// <param name="verbose">Whether to add metadata to the recognizer's results.</param>
-        /// <returns>The recognizer results and LUIS result.</returns>
-        private async Task<(RecognizerResult recognizerResult, LuisResult luisResult)> Recognize(LuisRequest request, CancellationToken ct, bool verbose)
+        private async Task<RecognizerResult> Recognize(LuisRequest request, CancellationToken ct, bool verbose)
         {
             var luisResult = await _luisService.QueryAsync(request, ct).ConfigureAwait(false);
-
             var recognizerResult = new RecognizerResult
             {
                 Text = request.Query,
-                AlteredText = luisResult.AlteredQuery,
+                AlteredText = luisResult.AlteredQuery ?? request.Query,
```

Comment:
This is interesting, and the semantics are a bit strange. This means it's up to me (the consumer) to determine if there's any altered text. A flag here, or even the semantics of leaving this as string.empt, might make more sense.

Reply:
Changed back to leave it null.  I'm also removing the text from the instance metadata as well.  Given the indices you can get the original string by doing (alteredtext ?? text).Substring(start, end).